pontoon/base/static/js/main.js

```diff
@@ -479,18 +479,19 @@ $(function() {
 
   // Menu sort
   $('.menu .sort span').click(function (e) {
-    function val(index, el) {
-      if (index !== 2) {
-        return $(el).find('span:eq(' + index + ')').html();
+    function getString(el) {
+      return $(el).find('span:eq(' + index + ')').html();
+    }
 
-      } else {
-        if (!$(el).find('.chart').length) {
-          return 0;
-        }
-        var chart = $(el).find('.chart').data('chart'),
-            data = JSON.parse(chart.replace(/'/g, "\""));
-        return data.approved/data.total;
+    function get(el, type) {
+      if (!$(el).find('.chart').length) {
+        return 0;
       }
+
+      var chart = $(el).find('.chart').data('chart'),
+          data = JSON.parse(chart.replace(/'/g, "\""));
```

Comment:
This is not really a good idea; you're relying on the happy accident that Python dicts are almost valid JSON. Instead, in the templates where you output the `data-chart` attribute, you should convert the dict to JSON. The upside is that `data()` automatically parses JSON so you won't have to call it here once that happens.